mat64/lq.go

```diff
@@ -210,7 +213,7 @@ func (v *Vector) SolveLQVec(lq *LQ, trans bool, b *Vector) error {
 	} else {
 		v.reuseAs(c)
 	}
-	m := vecAsDense(v)
-	bm := vecAsDense(b)
+	m := v.asDense()
```

Comment:
Considering moving these into the call line as `v.asDense().SolveLQ(lq, trans, b.asDense())`.